gpflow/kernels/multioutput/kernels.py

```diff
@@ -73,7 +73,9 @@ def K_diag(self, X, full_output_cov=True):
         """
         raise NotImplementedError
 
-    def __call__(self, X, X2=None, *, full_cov=False, full_output_cov=True):
+    def __call__(self, X, X2=None, *, full_cov=False, full_output_cov=True, presliced=False):
```

Comment:
Can you make sure that the parent class uses `presliced` option as well?

Reply:
What do you mean? The parent class is `Kernel` which supports preslicing.

Reply:
As noted separately, makes me think perhaps one shouldn't be a parent class of the other in the first place...